src/transports/graphql.ts

```diff
@@ -455,6 +569,38 @@ class TransportGraphQL implements TransportService {
     return this.doGraphQLMutation(query, variables, 'transaction');
   }
 
+  async UpdateTransactionMetadata(txID: string, metadata: Metadata): Promise<Transaction> {
+    const query = gql`
+      mutation ($tx_id: String!, $metadata: Metadata!) {
+        transaction_metadata (
+        tx_id: $tx_id
```

Comment:
I think it should be `id` vs `tx_id`

Reply:
@mrz1836 In graphql it is actually `tx_id`. We could change this, but that means a breaking change between clients and server

Reply:
Better to change it, it should indeed be `id`. Will update all repos